netbox/ipam/fields.py

```diff
@@ -32,6 +32,8 @@ def to_python(self, value):
     def get_prep_value(self, value):
         if not value:
             return None
+        if isinstance(value, list):
+            return [str(self.to_python(v)).split('/')[0] for v in value]
```

Comment:
I believe this is causing the filter to ignore prefix lengths. For example, assume I have an IP defined as 192.0.2.1/24. In the current release, both `address=192.0.2.1` and `address=192.0.2.1/24` will match this. However, `address=192.0.2.1/25` will not.

We need to preserve the ability to match the mask length, if specified.

Reply:
Solved this by the latest push.